frontend/src/components/House/styles.js

```diff
@@ -0,0 +1,21 @@
+import {StyleSheet} from 'aphrodite';
+
+const styles = StyleSheet.create({
+  list: {
+    display: "flex",
+    flexDirection: "column"
+  },
+  wrapper: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-around"
+  },
+  house_image: {
+    width: "200px"
+  },
+  title: {
+    textAlign: "center"
+  }
+});
+
+export default styles;
```

Comment:
I don't like this. But I guess each to their own. However I will say this, if we do this why don't we just stay with css... Afterall `class="img"` is shorter than `className={css(styles.img)}`

The whole point of aphrodite was that you can containerize and reuse your components. You can make them even more reuseable by having child components get passed down styles from their parent components, making it even more extensible and customiseable. My two cents anyway.

Reply:
I split the styles out as I needed one style in two files. Normally I use inline styles.

I think aphrodite allows us to cover the :hover css issue with react. This [video](https://vimeo.com/168648012) (from the react-boilerplate guys) has a good section on containerising css around 10:30.